dataprofiler/profilers/profiler_options.py

```diff
@@ -895,6 +905,34 @@ def _validate_helper(self, variable_path: str = "CategoricalOptions") -> list[st
                 "{}.top_k_categories must be either None"
                 " or a positive integer".format(variable_path)
             )
+
+        if self.max_sample_size_to_check_stop_condition is not None and (
+            not isinstance(self.max_sample_size_to_check_stop_condition, int)
+            or self.max_sample_size_to_check_stop_condition < 0
```

Comment:
why would a user specify a less than zero for `max_sample_size_to_check_stop_condition`?

Reply:
wouldn't it need to be `<=0`?

Reply:
current it would hit the `errors.append` if it was set to `0` but the error would say it must be non-negative which `0` is